src/map/packets_struct.h

```diff
@@ -3188,6 +3188,70 @@ struct PACKET_CZ_STYLE_CLOSE {
 DEFINE_PACKET_HEADER(CZ_STYLE_CLOSE, 0x0a48);
 #endif
 
+#if PACKETVER_MAIN_NUM >= 20161005 || PACKETVER_RE_NUM >= 20161005 || defined(PACKETVER_ZERO)
+struct PACKET_ZC_REFINE_OPEN_WINDOW {
+	int16 packetType;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_REFINE_OPEN_WINDOW, 0x0aa0);
+
+struct PACKET_CZ_REFINE_ADD_ITEM {
+	int16 packetType;
+	int16 index;
+};
+DEFINE_PACKET_HEADER(CZ_REFINE_ADD_ITEM, 0x0aa1);
```

Comment:
for 0x0aa1 because this is CZ packet not always possible say real minimal version usage. But client know this packet from this versions:
2016-10-05aRagexe
2016-10-05aRagexeRE
any zero